### DISCUSSION TITLE
WebSockets

### DISCUSSION THREAD

**Discussion by MasonSpeed** [livewire/livewire#1254] [4. General]
Are there any plans (Maybe in the **much** anticipated 2.0) to replace the Ajax implementation with a WebSocket/Pusher one?

**Comment by mblarsen** [comment]
I'm unclear as to the roadmap (is it public somewhere? [here](https://github.com/livewire/livewire/projects/2)), but Livewire is built with a network driver design where the default is a HTTP driver. So in theory it should be possible to add others like WebSockets.

Caleb did mention that he would look into for the future https://github.com/livewire/livewire/issues/874#issuecomment-652087269.

**Comment by calebporzio** [comment]
Good question, not on the roadmap for v2, but likely for the future.
